test-e2e/common/other.test.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+const path = require('path');
+
+const {promiseExec} = require('../test/helpers');
+
+const ESYCOMMAND = require.resolve('../../bin/esy');
+
+const packageJSON = require('../../package.json');
+
+it('Common - other', async () => {
+  expect.assertions(4);
+  const esy = args => promiseExec(`${ESYCOMMAND} ${args}`);
+
+  /* 
+   * TODO: 
+   * Should we use snapshots here?
+   * Can we reuse same snapshot for both help commmands?
+   */
+  const helpExpecting = expect.objectContaining({
```

Comment:
let's not test the output, these tests are just sanity checks, that commands like version, help don't fail